pgbouncer-mixin/alerts.libsonnet

```
@@ -0,0 +1,92 @@
+{
+  new(this): {
+
+    groups: [
+      {
+        name: this.config.uid,
+        rules: [
+          {
+            alert: 'HighNumberClientWaitingConnections',
+            expr: |||
+              pgbouncer_pools_client_waiting_connections > %(alertsHighClientWaitingConnections)s
+            ||| % this.config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'May indicate a bottleneck in connection pooling where too many clients are waiting for available server connections.',
+              description: |||
+                The number of clients waiting connections on {{ $labels.instance }} is now above %(alertsHighClientWaitingConnections)s. The current value is {{ $value | printf "%%.2f" }}.
```

```suggestion
                The number of client waiting connections on {{ $labels.instance }} is now above %(alertsHighClientWaitingConnections)s. The current value is {{ $value | printf "%%.2f" }}.
```
Or maybe this is "clients waiting for connections"? Up to you